python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -1164,6 +1169,9 @@ def test_latest_materialization_per_partition(self, graphql_context: WorkspaceRe
         asset_node = result.data["assetNodes"][0]
         assert len(asset_node["latestMaterializationByPartition"]) == 1
         materialization = asset_node["latestMaterializationByPartition"][0]
+        run_status = materialization["runOrError"]["status"]
+        assert run_status == "SUCCESS"
+
```

Comment:
You could verify here that the storage method is only called once, using a traced counter, like so:

https://github.com/dagster-io/dagster/blob/4683f1cc4850631f35aec1a04e83f12b99278a47/python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py#L1453

Reply:
done! that is very cool, did not know that was a thing